CSA.md

```diff
@@ -1,91 +1,93 @@
+      (b) <span class="coverpage_link">Customer's</span> use of the Product must comply with all Documentation and the <span class="coverpage_link">Acceptable Use Policy</span>, if any.
 
-    2.4 <span class="header_3">Suspension.</span>  If <span class="coverpage_link">Customer</span> (a) has an outstanding, undisputed balance on its account for more than 30 days after the <span class="orderform_link">Payment Period</span>, (b) breaches Section 2.1 (Restrictions on <span class="coverpage_link">Customer</span>), or (c) uses the Cloud Service in violation of the Agreement and in a way that materially and negatively impacts the Cloud Service or others, then <span class="coverpage_link">Provider</span> may temporarily suspend <span class="coverpage_link">Customer's</span> access to the Cloud Service with or without notice. However, <span class="coverpage_link">Provider</span> will try to inform <span class="coverpage_link">Customer</span> before suspending <span class="coverpage_link">Customer's</span> account when practical. <span class="coverpage_link">Provider</span> will reinstate <span class="coverpage_link">Customer's</span> access to the Cloud Service only if <span class="coverpage_link">Customer</span> resolves the underlying issue.
+    2.2 <span class="header_3">Suspension.</span>  If <span class="coverpage_link">Customer</span> (a) has an outstanding, undisputed balance on its account for more than 30 days after the <span class="orderform_link">Payment Period</span>, (b) breaches Section 2.1 (Restrictions on <span class="coverpage_link">Customer</span>), or (c) uses the Product in violation of the Agreement or in a way that materially and negatively impacts the Product or others, then <span class="coverpage_link">Provider</span> may temporarily suspend <span class="coverpage_link">Customer's</span> access to the Product with or without notice. However, <span class="coverpage_link">Provider</span> will try to inform <span class="coverpage_link">Customer</span> before suspending <span class="coverpage_link">Customer's</span> account when practical. <span class="coverpage_link">Provider</span> will reinstate <span class="coverpage_link">Customer's</span> access to the Product only if <span class="coverpage_link">Customer</span> resolves the underlying issue.
```

Comment:
Changing commas to semi colons, removing span class from Section 2.1 title reference, 

```suggestion
    2.2 <span class="header_3">Suspension.</span>  If <span class="coverpage_link">Customer</span> (a) has an outstanding, undisputed balance on its account for more than 30 days after the <span class="orderform_link">Payment Period</span>; (b) breaches Section 2.1 (Restrictions on Customer); or (c) uses the Product in violation of the Agreement or in a way that materially and negatively impacts the Product or others, then <span class="coverpage_link">Provider</span> may temporarily suspend <span class="coverpage_link">Customer's</span> access to the Product with or without notice. However, <span class="coverpage_link">Provider</span> will try to inform <span class="coverpage_link">Customer</span> before suspending <span class="coverpage_link">Customer's</span> account when practical. <span class="coverpage_link">Provider</span> will reinstate <span class="coverpage_link">Customer's</span> access to the Product only if <span class="coverpage_link">Customer</span> resolves the underlying issue.
```